hieradata/common.yaml

```diff
@@ -104,6 +104,9 @@ lookup_options:
   rke2::config:
     merge:
       strategy: "deep"
+  cni::plugins::enable:
+    merge:
+      strategy: "deep"
```

Comment:
What does this line of code do ?

Reply:
this means that if puppet finds more config on "that" on a different level of the hierarchy, it will not overwrite, but combine all the configuration together.